Dependency Management/Dependency Management.md

```diff
@@ -15,9 +56,27 @@ In order to keep repository size as minimal as possible, most of our apps do not
 * Primarily deal with sensitive data or financial transactions. For these apps, we will check in the `Pods` folder so that we can more easily audit the changes made to third party libraries as they are updated over time.
 * Require a non-BR build environment. Some client build environments don't allow network connections during the build process so all pod sources need to be present in the repository at build-time.
 
+### Understanding `pod` Commands
+
+Assuming you are using Bundler and have not installed Cocoapods globally all of these commands are used. If you **have** install Cocoapods globally simply use `pod <command>` instead of `bundle exec pod <command>`.
```

Comment:
Let me know if you think it's better to default to `bundle exec pod <command>` and have a note to remove `bundle exec` if Cocoapods is installed globally, or if we should default to `pod <command>` and have a note about prefixing it with `bundle exec` if you're using Cocoapods installed with Bundler

Reply:
I usually default to `bundle exec <command>` but I'm not sure if that's just me.